files/content/user_seed/init.lua

```diff
@@ -0,0 +1,94 @@
+--stylua: ignore start
+
+
+
+local allow_dev_mode = true --set to false to disable dev_mode
+
+
+
+
+RemoveFlagPersistent(("fairmod_developer_mode"):reverse())
```

Comment:
Here it uses `("fairmod_developer_mode"):reverse()` as the persistent flag but further down it uses `("edom_repoleved_domriaf"):reverse()`

Reply:
whoops